index.bs

```diff
@@ -9777,6 +9788,15 @@ The [=remote end steps=] given |session| and |command parameters| are:
 1. Let |arguments| be the <code>arguments</code> field of |command
    parameters| if present, or an empty [=/list=] otherwise.
 
+1. Let |user contexts| be [=set/create=] with |command parameters|[<code>userContexts</code>]
```

Comment:
nitpick: the rest of the spec seems to use `[=set/create|create a set=]` for set creation
```suggestion
1. Let |user contexts| be [=set/create|create a set=] with |command parameters|[<code>userContexts</code>]
```